.github/workflows/build.yml

```diff
@@ -90,19 +94,31 @@ jobs:
           name: ${{ steps.setup_precompiled.outputs.artifact_name }}
           path: ..
 
-      - name: tests
-        run: |
-          set -o pipefail
-          lean --make docs archive roadmap test | cat
-
       - name: lint
+        # for some reason this gets skipped if an earlier (non-"build") step fails,
+        # unless we include `success() ||`
```

Comment:
```suggestion
```
I understand why this happens now, it's the same reason the rest of the job gets skipped if the build step fails. By default, steps only run on `success() && <if-condition>` if there's no status check in the condition.